maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -148,6 +155,13 @@ fun Marker(
             set(flat) { this.marker.isFlat = it }
             set(icon) { this.marker.setIcon(it) }
             set(infoWindowAnchor) { this.marker.setInfoWindowAnchor(it.x, it.y) }
+            set(infoWindowVisibility) {
```

Comment:
This doesn't quite handle all the different edge cases. As currently written, this only handles updates to the Marker composable and not initialization.